digitalpy/core/telemetry/metrics.py

```diff
@@ -0,0 +1,4 @@
+from digitalpy.core.zmanager.controller import Controller
+
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>I003 isort expected 1 blank line in imports, found 0